schema/project-level/project-schema.json

```diff
@@ -2370,6 +2380,114 @@
         }
       },
       "minProperties": 1
+    },
+    "CostMeasurement": {
+      "title": "Cost measurement",
+      "description": "A measurement of costs for a project.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "title": "Local identifier",
+          "description": "A local identifier for the cost measurement.",
+          "type": "string",
+          "minLength": 1
+        },
+        "date": {
+          "title": "Date",
+          "description": "The date on which the cost measurement was reported.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "lifeCycleCosting": {
+          "title": "Life-cycle costing",
+          "description": "Information about life-cycle costing for the cost measurement.",
+          "type": "object",
+          "properties": {
+            "value": {
+              "title": "Life-cycle cost",
+              "description": "The life-cycle cost of the project.",
+              "$ref": "#/definitions/Value"
+            }
+          },
```

Comment:
I might have forgotten - Why does `lifeCycleCosting` have a `value` attribute, instead of being a `Value` itself? (in which case `lifeCycleCost` would be a more appropriate field name, title, etc)

Reply:
There are many other attributes relevant to life-cycle costing (the common date to which costs are discounted, the discount rate, the period of analysis etc.). CoST is only interested in the top-level value, but I wanted to leave a space for these other attributes in case a need for them comes up in the future.

For reference, here's the full list of life-cycle costing attributes from the RICS ICMS:

![image](https://github.com/open-contracting/infrastructure/assets/19265814/71090003-d89f-4c43-88f0-0d88d6d86c00)